utils/histogramming/hist_vars.py

```diff
@@ -0,0 +1,126 @@
+from hist.axis import Variable, Regular
```

Comment:
This should be something imported in config and should be stored either in configs/ or misc/

Reply:
moved to configs